eth/client.go

```diff
@@ -139,23 +140,23 @@ type client struct {
 	verifierAddr        ethcommon.Address
 	faucetAddr          ethcommon.Address
 
-	// Contract sessions
-	controllerSess          *contracts.ControllerSession
-	livepeerTokenSess       *contracts.LivepeerTokenSession
-	serviceRegistrySess     *contracts.ServiceRegistrySession
-	bondingManagerSess      *contracts.BondingManagerSession
-	ticketBrokerSess        *contracts.TicketBrokerSession
-	roundsManagerSess       *contracts.RoundsManagerSession
-	minterSess              *contracts.MinterSession
-	livepeerTokenFaucetSess *contracts.LivepeerTokenFaucetSession
+	// Contracts
+	controller          *contracts.Controller
```

Comment:
Like the refactor, but is this related to the timeouts change? Seems to be the bulk of the PR here.

Reply:
It is related because before adding the timeout, we actually needed to store sessions here (`CallOpts` was encapsulated in the session structs). Now, we don't need sessions, because we always provide our `CallOpts`.

Reply:
Cool! Thanks for clarification